tools/json_tools/generic_guns_validator.py

```diff
@@ -61,6 +61,9 @@
     "rebreather_cartridge_air",
     "rebreather_cartridge_o2",
     "rebreather_cartridge_regen"
+    'STI_DS_10'
```

Comment:
This would make it so that this gun didn't genericize.  You would need to follow the migration model to whatever the small pistol equivalent is for generic guns.

Reply:
I couldn't find that anywhere, so ill ask on discord

Reply:
done "i think"